Sources/Features/AssetsFeature/Components/PoolUnitsList/Components/LSUResource/LSUResource+View.swift

```diff
@@ -139,7 +76,7 @@ extension LSUResource.State {
 		.init(
 			isExpanded: isExpanded,
 			iconURL: .init(string: "https://i.ibb.co/KG06168/Screenshot-2023-08-02-at-16-19-29.png")!,
```

Comment:
is it correct that we hard code this iconURL (https://i.ibb.co/KG06168/Screenshot-2023-08-02-at-16-19-29.png)? it is low res and looks wrong...?

Reply:
hmm, why the hell we don't have a static resource for this...

Reply:
keeping as is, adding a TODO to remove it.